packages/botonic-plugin-flow-builder/src/action/index.tsx

```diff
@@ -68,19 +75,30 @@ export class FlowBuilderAction extends React.Component<FlowBuilderActionProps> {
   }
 
   render(): JSX.Element | JSX.Element[] {
-    const { contents } = this.props
+    const { contents, webchatSettingsParams } = this.props
     const request = this.context as ActionRequest
-    return contents.map(content => content.toBotonic(content.id, request))
+    return (
+      <>
+        {(isWebchat(request.session) || isDev(request.session)) &&
+          !!webchatSettingsParams && (
+            <WebchatSettings {...webchatSettingsParams} />
+          )}
+        {contents.map(content => content.toBotonic(content.id, request))}
+      </>
+    )
   }
 }
 
 export class FlowBuilderMultichannelAction extends FlowBuilderAction {
   render(): JSX.Element | JSX.Element[] {
-    const { contents } = this.props
-    const request = this.context as ActionRequest
+    const { contents, webchatSettingsParams } = this.props
+
     return (
       <Multichannel text={{ buttonsAsText: false }}>
-        {contents.map(content => content.toBotonic(content.id, request))}
+        <FlowBuilderAction
+          contents={contents}
+          webchatSettingsParams={webchatSettingsParams}
+        />
```

Comment:
Nit: This could be reduce to:
```suggestion
        <FlowBuilderAction { ...this.props } />
```

Reply:
applied